### PR TITLE
* [android] correct the format of the date returned by pickDate method.

### PR DESCRIPTION
**Pull request by coreycao**
The pickDate method in Android's Picker Module will return date like '2017-09-6' instead of '2017-09-06', this PR correct the problem.

### CODE REVIEW

**Comment by misakuo** [comment]
Thanks for your contribution, and please send this pr to branch `0.16-dev` :)

File: ios/sdk/WeexSDK/Sources/Component/WXWebComponent.m [hunk 1]
```diff
@@ -82,6 +82,7 @@ - (void)viewDidLoad
     _webview = (WXWebView *)self.view;
     _webview.delegate = self;
     _webview.allowsInlineMediaPlayback = YES;
+    _webview.scalesPageToFit = YES;
```
Comment (misakuo) [inline ios/sdk/WeexSDK/Sources/Component/WXWebComponent.m]: what is this?

Comment (coreycao) [reply ios/sdk/WeexSDK/Sources/Component/WXWebComponent.m]: @misakuo 

don't know, i just change the base branch to 0.16 - dev, this commit just appeared.

e83b0de is the last commit in dev branch, how to handle this situation ?

Comment (misakuo) [reply ios/sdk/WeexSDK/Sources/Component/WXWebComponent.m]: maybe you can revert it simply

Comment (coreycao) [reply ios/sdk/WeexSDK/Sources/Component/WXWebComponent.m]: How about close this PR, and create a new PR directly to 0.16-dev ?

Comment (misakuo) [reply ios/sdk/WeexSDK/Sources/Component/WXWebComponent.m]: Sure, you can close this and create new PR based on 0.16-dev

Comment (coreycao) [reply ios/sdk/WeexSDK/Sources/Component/WXWebComponent.m]: I see, thanks.

**Pull request closed without merging**